templates/starter/terraform/azure.yaml

```diff
@@ -0,0 +1,32 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/Azure/azure-dev/main/schemas/v1.0/azure.yaml.json
+
+# This is an example starter azure.yaml file containing several example services.
+# Make changes as needed to describe your application setup.
+# To learn more about the azure.yaml file, visit https://learn.microsoft.com/en-us/azure/developer/azure-developer-cli/azd-schema
+
+# Name of the application.
+name: azd-starter
+services:
+  # An example for a python API service.
+  # The service is named 'python-api'.
+  # The language is 'python'.
+  # The source code is located in the project (azure.yaml) directory.
+  # The service will be hosted on Azure App Service.
+  python-api:
+    language: python
+    project: ./
```

Comment:
same comment about code in root

Reply:
@savannahostrowski What is idiomatic for python?